src/polymer/feature-finder.ts

```diff
@@ -43,14 +45,15 @@ export function featureFinder() {
       return;
     }
 
-    for (const prop of featureNode.properties.map(
-             esutil.toScannedPolymerProperty)) {
+    const polymerProps = featureNode.properties.map(
+        p =>
```

Comment:
style nit: I prefer always using parens on arrow functions (it's also a "tip" int he style guide)